app/javascript/styles/mastodon-light/diff.scss

```diff
@@ -675,7 +675,6 @@ html {
   .public-account-bio,
   .hero-widget__text {
     background: $account-background-color;
-    border: 1px solid lighten($ui-base-color, 8%);
```

Comment:
You just need to set `border-bottom: 0`, I think. Just one side, not all of them.

Reply:
This seems rather hacky and I'm not sure if it will even have an effect. On my side it kinda seems to want the whole border removed as the border styling has been done something like this

![image](https://user-images.githubusercontent.com/40538332/73205069-85fec480-4140-11ea-8f59-598dd6e6d579.png)

So no matter what I do removing the border completely does the job